pkg/api/customization/cluster/actionhandler.go

```diff
@@ -525,3 +531,56 @@ func (a ActionHandler) RotateCertificates(actionName string, action *types.Actio
 	http.ServeContent(apiContext.Response, apiContext.Request, "rotate", time.Now(), reader)
 	return nil
 }
+
+func (a ActionHandler) BackupEtcdHandler(actionName string, action *types.Action, apiContext *types.APIContext) error {
+	cluster, err := a.ClusterClient.Get(apiContext.ID, metav1.GetOptions{})
+	if err != nil {
+		return err
+	}
+
+	newBackup := etcdbackup.NewBackupObject(cluster)
+	response := map[string]interface{}{
+		"backup": newBackup.Name,
```

Comment:
should we be adding this only if create is successful?

Reply:
`newBackupObject()` just builds the object, it doesn't actually create anything on the API.